test/nuts/create.nut.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { join } from 'path';
+import { expect } from '@salesforce/command/lib/test';
+import { TestSession } from '@salesforce/cli-plugins-testkit';
+import { execCmd } from '@salesforce/cli-plugins-testkit';
+import { fs } from '@salesforce/core';
+import { Dictionary } from '@salesforce/ts-types';
+
+const apexManifest =
+  '<?xml version="1.0" encoding="UTF-8"?>\n' +
+  '<Package xmlns="http://soap.sforce.com/2006/04/metadata">\n' +
+  '    <types>\n' +
+  '        <members>GeocodingService</members>\n' +
+  '        <members>GeocodingServiceTest</members>\n' +
+  '        <members>PagedResult</members>\n' +
+  '        <members>PropertyController</members>\n' +
+  '        <members>SampleDataController</members>\n' +
+  '        <members>TestPropertyController</members>\n' +
+  '        <members>TestSampleDataController</members>\n' +
+  '        <name>ApexClass</name>\n' +
+  '    </types>\n' +
+  '    <version>52.0</version>\n' +
```

Comment:
Unless you override the API version this will fail as soon as 52.0 is not current.